### PR TITLE
Add example for networkxd3forcelayout plugin

### PR DESCRIPTION
**Pull request by kdheepak**

### CODE REVIEW

File: examples/networkxd3forcelayout.py [hunk 1]
```diff
@@ -0,0 +1,285 @@
+"""
+
+NetworkX to d3.js Force Layout
+==============================
+
+MPLD3 Plugin to convert a NetworkX graph to a force layout.
+This is an example demoed `here
+<http://blog.kdheepak.com/mpld3-networkx-d3js-force-layout.html>`_
+You can download the plugin from the Github repo `here
+<https://github.com/kdheepak/mpld3_plugins/blob/master/mpld3_plugins/plugins/networkxd3forcelayout.py>`_
```
Comment (aflaxman) [inline examples/networkxd3forcelayout.py]: ah, my mistake for reading too quickly.  i see now, and this is awesome.

**Comment by kdheepak** [comment]
Great! Thanks!

**Comment by kdheepak** [comment]
Just curious, will this update the examples page on mpld3.github.io as well?

**Comment by aflaxman** [comment]
I bet I need to rebuild that manually. @jakevdp do you remember how to do such a thing?

**Comment by etgalloway** [comment]
The examples are updated automatically when the website is rebuilt. RELEASING.md has the step-by-step instructions for doing the build.

**Pull request merged**